File: data/readable_format/fortran/file016.f90
program trivialTest
  implicit none
  call sayHello()
  print *, "The square of 5 is ", square(5)
contains

  subroutine sayHello()
    print *, "Hello, World!"
  end subroutine sayHello

  function square(x) result(res)
    implicit none
    integer, intent(in) :: x
    integer :: res
    res = x * x
  end function square

end program trivialTest

File: data/readable_format/cpp/file016.cpp
#include <iostream>

// Declaration of the function to print Hello, World!
void sayHello() {
    std::cout << "Hello, World!" << std::endl;
}

// Declaration and definition of the function to square a number
int square(int x) {
    return x * x;
}

int main() {
    sayHello();
    std::cout << "The square of 5 is " << square(5) << std::endl;
    return 0;
}